packages/arb-bridge-eth/contracts/rollup/facets/RollupAdmin.sol

```diff
@@ -242,24 +242,29 @@ contract RollupAdminFacet is RollupBase, IRollupAdmin {
         emit OwnerFunctionCalled(20);
     }
 
-    /*
-    function forceResolveChallenge(address[] memory stackerA, address[] memory stackerB) external override whenPaused {
-        require(stackerA.length == stackerB.length, "WRONG_LENGTH");
-        for (uint256 i = 0; i < stackerA.length; i++) {
-            address chall = inChallenge(stackerA[i], stackerB[i]);
+    function forceResolveChallenge(address[] memory stakerA, address[] memory stakerB)
+        external
+        override
+        whenPaused
+    {
+        require(stakerA.length == stakerB.length, "WRONG_LENGTH");
+        for (uint256 i = 0; i < stakerA.length; i++) {
+            address chall = inChallenge(stakerA[i], stakerB[i]);
 
             require(address(0) != chall, "NOT_IN_CHALL");
-            clearChallenge(stackerA[i]);
-            clearChallenge(stackerB[i]);
+            clearChallenge(stakerA[i]);
+            clearChallenge(stakerB[i]);
 
             IChallenge(chall).clearChallenge();
         }
+        emit OwnerFunctionCalled(21);
     }
 
-    function forceRefundStaker(address[] memory stacker) external override whenPaused {
-        for (uint256 i = 0; i < stacker.length; i++) {
-            withdrawStaker(stacker[i]);
+    function forceRefundStaker(address[] memory staker) external override whenPaused {
+        for (uint256 i = 0; i < staker.length; i++) {
+            withdrawStaker(staker[i]);
         }
+        emit OwnerFunctionCalled(22);
     }
 
     function forceCreateNode(
```

Comment:
It'd be good to talk through the uses of this function relative to standard node creation by a validator

Reply:
I added in a comment in these that make explicit the validations that are skipped